src/content/Concerts/12-13Soloist.md

```diff
@@ -0,0 +1,43 @@
+---
+# Concert Name (e.g. "Fall Concert")
+concertName: "Soloist Spotlight"
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2013-03-09"
+# Season
+season: "2012/13"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
+# Google Calendar link (if applicable)
+calendar: ""
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Tristan und Isolde
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 8, 2013 19:30
+Datetime = March 9, 2013 19:30
+
+Section
+
+Overline = Program
+Work = J. S. Bach ~ Piano Concerto No. 5 in F minor, DWV. 1056 ~ Ruth Ochs, Guest Conductor ~ Paul von Autenreid ’16, Piano
```

Comment:
Don't add guest conductor. See 2016/17 season

Reply:
Also fix the order (conductor always goes second for concerti)

Reply:
tried and fixed it don't know if it's right. 16/17 season had no guest conductor only guest groups